aloha/robot_utils.py

```diff
@@ -21,154 +18,81 @@
 class ImageRecorder:
     def __init__(
         self,
-        is_mobile: bool = IS_MOBILE,
+        config: dict,
         is_debug: bool = False,
         node: Node = None,
     ):
         self.is_debug = is_debug
         self.bridge = CvBridge()
 
-        if is_mobile:
-            self.camera_names = ['cam_high', 'cam_left_wrist', 'cam_right_wrist']
-        else:
-            self.camera_names = ['cam_high', 'cam_low', 'cam_left_wrist', 'cam_right_wrist']
+        # Get camera names from config dictionary
+        self.camera_names = [camera['name'] for camera in config.get('cameras', {}).get('camera_instances', [])]
+
+        COLOR_IMAGE_TOPIC_NAME = config.get('cameras',{}).get('common_parameters', {}).get('color_image_topic_name', None)
```

Comment:
No longer a const - make snake_case

```suggestion
        color_image_topic_name = config.get('cameras',{}).get('common_parameters', {}).get('color_image_topic_name', None)
```